packages/plugin-types/API.md

```diff
@@ -39,8 +39,18 @@ export type AuthResources = {
     cfnResources: AuthCfnResources;
 };
 
-// @public (undocumented)
-export type BackendId = AppId | SandboxId;
+// @public
+export type BackendIdentifier = {
+    namespace: Readonly<AppId>;
+    name: Readonly<BranchName>;
+    type: Readonly<'branch'>;
+    hash?: Readonly<string>;

```

Comment:
hehe, this is where `disambiguator` wouldn't be bad naming ;-)

Reply:
Lol I had a similar thought but I'm scarred from the name so I went with hash.